docs/diagrams/container.puml

```diff
@@ -21,10 +21,10 @@ Boundary(aws_cloudplatform, "Cloud Platform", "A Kubernetes cluster using Amazon
     Boundary(upstream, "Legacy Upstream Systems", "Various HMPPS upstream systems, some of these will eventually be replaced"){
         Container(prison_api, "Prison API", "Spring Boot Java API", "API over NOMIS DB used by Digital Prison team applications and services", $link="https://github.com/ministryofjustice/prison-api")
         Container(probation_offender_search, "Probation Offender Search", "Springoot Kotlin API", "API over the nDelius offender data held in Elasticsearch", $link="https://probation-offender-search-dev.hmpps.service.justice.gov.uk/swagger-ui/index.html")
-        Container(prison_offender_search, "Prisoner Offender Search", "Spring Boot Kotlin API", "API over the NOMIS prisoner data held in Elasticsearch", $link="https://github.com/ministryofjustice/prisoner-offender-search")
+        Container(prisoner_search, "Prisoner Search", "Spring Boot Kotlin API", "API over the NOMIS prisoner data held in Elasticsearch", $link="https://github.com/ministryofjustice/hmpps-prisoner-search")
```

Comment:
note that the data these days is held in OpenSearch instead of elastic search

Reply:
Ahhhhh, thanks Peter for this spot!

Reply:
PR created to fix dis: https://github.com/ministryofjustice/hmpps-integration-api/pull/280.